File: jiyeong/week_01/CODETREE_배수찾기.py
N = int(input())

arr = [3, 5, 3, 3, 5, 3, 3]
nums = [3, 5, 6, 9, 10, 12, 15]
idx = N%7 - 1
num = arr[idx]
m = N//7
if N % 7 == 0:
    m -= 1
# print(idx)
print(15*m+nums[idx])

File: hanbin/week_01/CODETREE_배수 찾기.js
// 풀이 전략: 규칙 찾아서 O(1)에 처리

const N = Number(require('fs').readFileSync(0).toString().trim());

const arr = [0, 3, 5, 6, 9, 10, 12, 15];

let idx  = N % 7;
let mult = Math.floor(N / 7);
console.log(arr[idx]+ 15 * mult);
